### PR TITLE
HOCS-2272 Missing forename/surname of users in MUI shows up as "null"

### PR DESCRIPTION
**Pull request by LaurieBennett**
On the team management screen and other applicable screens:

    a blank forename in Keycloak should not show up as "null" in the management UI
    a blank surname in Keycloak should not show up as "null" in the management UI
    a blank email in Keycloak should not show up as "null" in the management UI
    a profile with neither forename or surname shouldn't have a bracketed email address

For example,
null null (foo@example.com)
should read

foo@example.com

and

Jane null (foo@example.com)

should read

Jane (foo@example.com) 

But a user actually called "Jane Null" should have their name show up as usual.

### CODE REVIEW

File: server/lists/adapters/__tests__/users.spec.js [hunk 1]
```diff
@@ -21,4 +21,24 @@ describe('User Adapter', () => {
...
         expect(results).toMatchSnapshot();
     });
+
+    it('should correctly transform null values', async () => {
+        const mockData = [
+            { firstName: null, lastName: 'A', email: 'user.a@example.com', id: 1 },
+            { firstName: 'User', lastName: null, email: 'user.b@example.com', id: 2 },
+            { username: 'userC', firstName: 'User', lastName: 'C', email: null, id: 3 },
+            { username: 'userD', firstName: null, lastName: null, email: 'user.D@example.com', id: 4 },
+            { username: 'userE', firstName: null, lastName: null, email: null, id: 5 }
```
Comment (robertdeniszczyc2) [inline server/lists/adapters/__tests__/users.spec.js]: I think there are maybe some other possible scenarios here:
- User has a username, firstname, email, but lastname null 
- User has a username, lastname, email , but firstname null
- User has a username, firstname, but lastname and email null
- User has a username, lastname, but firstname and email null
- User has a first name or last name of "null" eg "Jane Null"

**Review by robertdeniszczyc2 — COMMENTED** [review]
Added a comment around possible extra test cases to add to unit test

File: server/lists/adapters/__tests__/users.spec.js (hunk 1 above)
Comment (LaurieBennett) [reply server/lists/adapters/__tests__/users.spec.js]: Done, I've added a username to most of the data and covered some of the missing potential states.

**Review by robertdeniszczyc2 — APPROVED** [review]
Looks good to me

**Pull request merged**